docs/assets/templates/.dockstore.yml

```diff
@@ -8,13 +8,15 @@ service:
   name:
 
   # Optional array of authorship information, requires at least the name of each author.
+  # Note that just using "author" as a singular string is deprecated as of 1.2
```

Comment:
I wouldn't bother mentioning this; I just checked and we don't have any version 1.1 .dockstore.yml

Reply:
It is in assets/templates. Is that intentional? I don't think it gets rendered anywhere on our site, but it's still kicking around.

Reply:
Ah, I see. Yeah, I don't see any references to .dockstore-1-1.yml, so I think you can still drop this.